pypeit/core/wavecal/waveio.py

```diff
@@ -177,6 +179,25 @@ def load_source_table():
     return sources
 
 
+def load_tree(polygon=4, numsearch=20):
+    import pickle
```

Comment:
Do we have to save as a `pickle`?

If we are multi-processing the answer can be yes.
Otherwise think twice.

Reply:
Hmm... I'm not entirely sure, but I couldn't think of any other way to save the tree structure! Googling around online, it seems that this is the standard.

Reply:
ok, a pickle we go.